File: CS/MyTextEditWithCustomAutocomplete/MyAutoCompleteMaskManager.cs
using System;
using DevExpress.Data.Mask;

namespace AutoCompleteMaskManager {
    public class MyAutoCompleteMaskManager : MaskManagerSelectAllEnhancer<MaskManagerCommon<MaskManagerPlainTextState>> {
        public MyAutoCompleteMaskManager(MaskManagerCommon<MaskManagerPlainTextState> nested)
            : base(nested) {
        }
    }
}

File: VB/MyTextEditWithCustomAutocomplete/MyAutoCompleteMaskManager.vb
Imports System
Imports DevExpress.Data.Mask

Namespace AutoCompleteMaskManager
	Public Class MyAutoCompleteMaskManager
		Inherits MaskManagerSelectAllEnhancer(Of MaskManagerCommon(Of MaskManagerPlainTextState))

		Public Sub New(ByVal nested As MaskManagerCommon(Of MaskManagerPlainTextState))
			MyBase.New(nested)
		End Sub
	End Class
End Namespace
